### PR TITLE
Updated the Readme to include Background Replacement notice

### PR DESCRIPTION
**Pull request by jhaarpan**
**Issue #:**

**Description of changes:**
Added the notice for Background Replacement in the Readme Notice section.

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*

**Checklist:**

1. Have you successfully run `npm run build:release` locally? Yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1253,6 +1253,8 @@ The use of Amazon Voice Focus and background blur via this SDK involves the down
 
 The use of Amazon Voice Focus and background blur runtime code is subject to additional notices. See [this Amazon Voice Focus NOTICES file](https://static.sdkassets.chime.aws/workers/NOTICES.txt) and [background blur NOTICES file](https://static.sdkassets.chime.aws/bgblur/workers/NOTICES.txt) for details. You agree to make these additional notices available to all end users who use Amazon Voice Focus and background blur runtime code via this SDK.
 
+You and your End Users are responsible for all Content (including any images) uploaded for use with video background replacement, and must ensure that such Content does not violate the law, infringe or misappropriate the rights of any third party, or otherwise violate a material term of your agreement with Amazon (including the documentation, the AWS Service Terms, or the Acceptable Use Policy).
```
Comment (ltrung) [inline README.md:1256]: Do we need to capitalize End Users and Content?

Comment (jhaarpan) [reply README.md:1256]: This was given to us by the legal team

**Comment by ltrung** [comment]
Need to resolve conflict in README.

**Review by ltrung — DISMISSED** [review]

**Comment by michhyun1** [comment]
need to run `npm run build:release`

**Review by michhyun1 — APPROVED** [review]

**Review by michhyun1 — APPROVED** [review]

**Pull request merged**